packages/babel-plugin/__tests__/transform-stylex-create-test.js

```diff
@@ -1302,12 +1352,32 @@ describe('@stylexjs/babel-plugin', () => {
               }
             });
           `);
-          expect(code).toMatchInlineSnapshot();
-          expect(metadata).toMatchInlineSnapshot();
+          expect(code).toMatchInlineSnapshot(`
+            "import * as stylex from '@stylexjs/stylex';
+            export const styles = {
+              root: {
+                kMwMTN: "x8vgp76",
+                $$css: true
+              }
+            };"
+          `);
+          expect(metadata).toMatchInlineSnapshot(`
+            {
+              "stylex": [
+                [
+                  "x8vgp76",
+                  {
+                    "ltr": ".x8vgp76{color:var(--color,red);color:color-mix(in srgb,currentColor 20%,transparent)}",
+                    "rtl": null,
+                  },
+                  3000,
+                ],
+              ],
+            }
+          `);
         });
 
-        // TODO: Fix parser bug
-        test.skip('args: func, var, value, value', () => {
+        test('args: func, var, value, value', () => {
```

Comment:
Thinking about this one. The assumptions originally made here might not be correct. For example, there's no fallback if variables aren't supported.

Reply:
We intentionally didn't implement a fallback for browsers where variables aren't supported as they're very widely supported in all browsers. Given that it would increase the CSS size a bit, do you think it's worth doing or doing conditionally based on a compiler config?